### PR TITLE
build

### PR DESCRIPTION
**Pull request by ckahub**
#Rating
To using the rating in the above example, we can use the following code with rating class
rating
.

### CODE REVIEW

File: Components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@import url('../../styles/utility.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+
+.btn{
+    font-weight: 500;
+    background: var(--primary-color);
+    color: white;
```
Comment (prakashsakari) [inline Components/Modal/modal.css:9]: use hex code for color for consistency

**Review by prakashsakari — APPROVED** [review]
Looks good to me

File: Components/Rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ranitasahaa1230) [inline Components/Rating/rating.html:6]: You can add desc. in the meta tag like author name, page desc, page content etc

File: Components/Rating/rating.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                    <li> 
+                        <a class="active" href="/Components/Modal/modal.html">
+                        Modal
+                         </a>
+                    </li>
+                 </ul>
+                 <ul class="sub_Topic">
+                    <li> 
+                        <a class="active" href="/Components/Rating/rating.html">
+                        Ratting
```
Comment (ranitasahaa1230) [inline Components/Rating/rating.html:101]: Spelling mistake->it should be Rating

**Review by ranitasahaa1230 — APPROVED** [review]
Overall, looks good to me.

**Review by HeyNitin — APPROVED** [review]
Please make changes as suggested by peers, rest LGTM

**Pull request merged**